### PR TITLE
feat: cors accelerator

### PR DESCRIPTION
**Pull request by apigeek3000**

### CODE REVIEW

File: cors/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+Apigee's [CORS policy](https://cloud.google.com/apigee/docs/api-platform/reference/policies/cors-policy) offers a simple CORS solution for your APIs. Consider this policy if you desire to call your Apigee proxy from a different domain. CORS policies can also allow/deny other request parameters such as the method or header.
+
+## Implementation on Apigee 
+
+The Apigee proxy sample is a no-target proxy with only a single policy; a CORS policy that allows access for all cross-origin domains
+
+## Prerequisites
+1. [Provision Apigee X](https://cloud.google.com/apigee/docs/api-platform/get-started/provisioning-intro)
+2. Configure [external access](https://cloud.google.com/apigee/docs/api-platform/get-started/configure-routing#external-access) for API traffic to your Apigee X instance
+3. Have access to deploy proxies, create products, and provision a portal in Apigee
```
Comment (drush1980) [inline cors/README.md:18]: This sample doesn't require API products or a portal

File: cors/deploy-cors.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+# var is expected by integration test (apickli)
+export PROXY_URL="$APIGEE_HOST/v1/samples/cors"
+
+echo " "
+echo "All the Apigee artifacts are successfully deployed!"
+
+echo " "
+echo "Your Proxy URL is: https://$PROXY_URL"
+echo "You will be unable to call the proxy url until you create an API key from the developer portal"
```
Comment (drush1980) [inline cors/deploy-cors.sh:54]: Remove this line

File: cors/deploy-cors.sh [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+echo "Please folow on with the instructions to create an API Key"
```
Comment (drush1980) [inline cors/deploy-cors.sh:55]: And this one

File: cors/docs/cloudshell-tutorial.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+Now that our API proxy is deployed, let's enable the debugger so we can see requests as they come through:
+1. Navigate to the [Apigee homepage](https://apigee.google.com). Go to Develop > API Proxies and click into the sample-cors proxy
+2. Navigate to the debug tab
+3. Choose to start a debug session for the currently deployed proxy version
+
+With the proxy debugger still running, call the sample CORS proxy from a cross-origin client:
+1. Navigate to an online HTTP request service such as [test-cors.org](https://test-cors.org). The requests need to be sent from an online service and should not be sent from your local machine.
+2. From [test-cors.org](https://test-cors.org) find the Remote URL field and enter in your proxy's URL, https://\[APIGEE_HOST\]/v1/samples/cors. Leave all other fields at their default values
+3. Click the Send Request button and view the response. You should see 200 response, meaning our CORS policy is working as expected and our test was a success!
+4. Navigate to the Apigee debugger. You should see a 200 response there as well!
```
Comment (drush1980) [inline cors/docs/cloudshell-tutorial.md:57]: Maybe it's worth showing an example of a blocked request as well.  What do you think?

Comment (apigeek3000) [reply cors/docs/cloudshell-tutorial.md:57]: I thought about this, and talked about it with some others as well. The decision we came to was to keep the accelerator as short and sweet as possible. Perhaps I could add a "Optional: Trigger CORS Error" section at the end?

Comment (drush1980) [reply cors/docs/cloudshell-tutorial.md:57]: If you're going to add a failed request, I think it should come before the successful one.  I think having both a negative and a positive result helps to demonstrate that the policy is really doing what it's supposed to.

**Review by drush1980 — APPROVED** [review]
lgtm

**Pull request merged**